openid-sharedsignals-framework-1_0.md

```diff
@@ -957,6 +958,11 @@ description
   This is useful in multi stream systems to identify the stream for human actors. The
   transmitter may truncate the string beyond allowed max length.
 
+receiver_key
```

Comment:
1. We are bringing in receiver-specific wording in the object, which is very stream-specific
2. We should specifically call the key `*_jwks_url` to align with the Tx metadata key and disambiguate its purpose.

Reply:
For more info, should we also reference the `Encrypting SETs` section?